controllers/glanceapi_controller.go

```diff
@@ -642,6 +670,11 @@ func (r *GlanceAPIReconciler) reconcileNormal(ctx context.Context, instance *gla
 				err.Error()))
 			return ctrlResult, err
 		} else if (ctrlResult != ctrl.Result{}) {
+			instance.Status.Conditions.MarkFalse(
```

Comment:
nit: I would consider this as Unknown instead of False based on the last bulletpoint of https://github.com/openstack-k8s-operators/docs/blob/main/conditions.md#conditions-in-openstack-k8s-operators-podified-controlplane-operators

Reply:
Based on this code, `instance.Spec.TLS.CaBundleSecretName` is not empty, `ValidateCACertSecret` fails, and I assume there's something that should be changed somewhere in the CR (or the openstack-operator), so I'm not sure that the glance-operator has the logic to make this progressing to a `Status=True` state without any "external" or "user" related intervention: am I interpreting this scenario wrong?

Reply:
I had a comment on [1], but if it meets the definition of `Unknown` I'm ok to change it to `Unknown`. Also this is not an `error` situation, we're comparing `ctrlResult != ctrl.Result{}` but actually no error is returned, so I think it makes sense to move it to use `Unknown` instead of `False`. Comparing `ctrlResult != ctrl.Result{}` feels like something still in progress rather than an error that should mark the condition to `False` and return an `err`.
Thank you, this is a good catch! I'll change that in my next iteration.

[1] https://github.com/openstack-k8s-operators/docs/pull/99